Bio/Cluster/__init__.py

```diff
@@ -42,10 +42,16 @@
 
 
 class Node(_cluster.Node):
+    """Define an element of a hierarchical clustering tree.
+
+    A node contains items or other Nodes(sub-nodes)."""
     __doc__ = _cluster.Node.__doc__
 
 
 class Tree(_cluster.Tree):
+    """Define methods to support a hierarchical clustering tree.
```

Comment:
Why "Define"? I would think just "Hierarchical clustering tree." is better here.

Reply:
Agree